arduino-ide-extension/src/browser/theia/core/window-title-updater.ts

```diff
@@ -1,18 +1,28 @@
+import { nls } from '@theia/core';
```

Comment:
expand it

Reply:
https://github.com/arduino/arduino-ide/pull/1901/commits/6471e1dc0813a1309da9b6b69d9bff378e05a0b3